pom.xml

```diff
@@ -98,8 +98,8 @@
     <javassist.version>3.29.2-GA</javassist.version>
     <javax.el.version>3.0.0</javax.el.version>
     <javax.validation.api>2.0.1.Final</javax.validation.api>
-    <jersey.version>2.40</jersey.version>
-    <jetty.version>9.4.56.v20240826</jetty.version>
+    <jersey.version>3.1.9</jersey.version>
+    <jetty.version>12.0.15</jetty.version>
```

Comment:
btw, Jetty has BOM dependencies that can be used as well.
One for core, and one for the environment you choose to use.
Would you be interested in that?